t/full-stack.t

```diff
@@ -191,9 +191,8 @@ $driver->get($job_page_url);
 ok wait_for_result_panel($driver, qr/Result: passed/), 'test 1 is passed' or show_job_info 1;
 my $autoinst_log = autoinst_log(1);
 ok -s $autoinst_log, 'autoinst log file generated';
-my $worker_log = $autoinst_log->dirname->child('worker-log.txt');
-ok -s $worker_log, 'worker log file generated';
-my $log_content = $worker_log->slurp;
+my $worker_log  = $autoinst_log->dirname->child('worker-log.txt');
+my $log_content = eval { $worker_log->slurp };
```

Comment:
This way `$log_content` would end up as undef in case of an error, so we would get a unitialized warning in the next test